emails/models.py

```diff
@@ -223,10 +223,12 @@ def address_default():
 
 
 def has_bad_words(value):
-    return any(
-        badword in value
-        for badword in emails_config.badwords
-    )
+    for badword in emails_config.badwords:
+        if len(badword) <= 3 and badword == value:
+            return True
```

Comment:
Potentially very irrelevant, but while looking at the /emails/badwords.txt list, I noticed a few of them have trailing whitespace. I don't know django, but just wanted to confirm that somewhere is trimming the bad word list before comparing word lengths. Although I think only one of the results is 3 characters (but 4 with trailing space) and might have a slightly different behavior if trimmed or not.

```sh
cat emails/badwords.txt | grep -e "\s$"
```